kodiak/config.py

```diff
@@ -57,6 +57,8 @@ class Merge(BaseModel):
     method: MergeMethod = MergeMethod.merge
     # delete branch when PR is merged
     delete_branch_on_merge: bool = False
+    # deprecated. this feature is flawed and cannot be fixed. see https://github.com/chdsbd/kodiak/issues/153#issuecomment-523057332.
```

Comment:
Maybe write DEPRECATED in upper case or on a separate line, so it will stand out?